src/chrome/content/rules/Hollywood-Reporter.xml

```diff
@@ -37,26 +33,17 @@
 
 	<target host="hollywoodreporter.com" />
 	<target host="*.hollywoodreporter.com" />
-		<!--
-			Avoid user-visible paths:
-							-->
-		<exclusion pattern="^http://(?:www\.)?hollywoodreporter\.com/(?!favicon\.ico|sites/)" />
-		<!--
-			css_c602cf5d0b126628beb7012985fd4591 links to images relative /
-											-->
-		<exclusion pattern="^http://(?:www\.)?hollywoodreporter\.com/sites/default/files/css/(?!css_02b3ba95a2adca47bf1e40d170fcaa56\.css)" />
-
 
 	<securecookie host=".+\.hollywoodreporter\.com$" name=".+" />
 
+    <test url="http://www.hollywoodreporter.com/" />
+    <test url="http://subscribe.hollywoodreporter.com/" />
 
-	<rule from="^http://(?:www\.)?hollywoodreporter\.com/"
-		to="https://a248.e.akamai.net/f/1682/7219/8m/www.hollywoodreporter.com/" />
 
-	<rule from="^http://m\.hollywoodreporter\.com/mobify/"
-		to="https://hollywoodreporter.mobify.com/mobify/" />
+	<rule from="^http://(?:www\.)?hollywoodreporter\.com/"
+		to="https://www.hollywoodreporter.com/" />
```

Comment:
^hollywoodreporter.com seems to work. We prefer not to 'snap' redirects if possible, since doing so is both more complex and more distinguishing.